opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -178,6 +178,13 @@ message ScopeProfiles {
   // https://opentelemetry.io/docs/specs/otel/schemas/#schema-url
   // This schema_url applies to all profiles in the "profiles" field.
   string schema_url = 3;
+
+  // The preferred type and unit of Samples in at least one Profile.
+  // For a cpu or off-cpu profile this might be:
```

Comment:
```suggestion
  // Also used as fallback for Profiles with missing sample_type.
  // For a cpu or off-cpu profile this might be:
```

Reply:
As Profile.sample_type is not optional, naming this field as fallback sounds confusing and conflicts in some way.